.pre-commit-config.yaml

```diff
@@ -9,24 +9,15 @@ repos:
   - id: debug-statements
   - id: mixed-line-ending
     args: [--fix=lf]
-- repo: https://github.com/pycqa/isort
-  rev: 5.12.0
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: v0.0.254
   hooks:
-  - id: isort
-    args: [--profile=black]
-- repo: https://github.com/asottile/pyupgrade
-  rev: v3.3.1
-  hooks:
-  - id: pyupgrade
-    args: [--py36-plus]
+  - id: ruff
+    args: [--format, github]  # --fix, --exit-non-zero-on-fix]
```

Comment:
Maybe args are not needed here when the same is specified in the configuration? Having that specified just once is probably a better approach.

Reply:
If your run `ruff` on the command line or in `pre-commit` then you do not want `--format=github` but if you run it in GitHub Actions then you do.  So, put the majority of config in `pyproject.toml` and only `--format` in args.

Reply:
But this does not address only running inside github action, this is generic pre-commit configuration...